lib/internal/bootstrap/loaders.js

```diff
@@ -178,17 +185,31 @@ let internalBinding;
   };
 }
 
+const getOwn = (target, property, receiver) => {
+  return ObjectPrototypeHasOwnProperty(target, property) ?
+    ReflectGet(target, property, receiver) :
+    undefined;
+};
```

Comment:
This feels like a general-purpose helper that we either probably already have somewhere else or should be putting in some central place.

Reply:
This is already here before the PR though, I think I should fix up to make git stop treating it as a change instead..

Reply:
Fixed up to make git ignore this bit.